src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css

```diff
@@ -52,3 +52,9 @@
 	background-repeat: no-repeat;
 	padding: 2px;
 }
+
+.accessible-view.hide {
+	position: fixed;
+	top: -2000px;
+	left:-2000px;
```

Comment:
Does `visibility: hidden` work? Some things can go wrong when you put things off screen

Reply:
nothing went wrong when I tested it, but I'll try that

Reply:
does not work with `visibility:hidden`, focus remains in the terminal

Reply:
Maybe shld try opacity 0